.github/workflows/ci.yaml

```diff
@@ -33,6 +33,15 @@ jobs:
       - name: Check links
         run: npm run check-links
 
+      # Share data between the build and deploy jobs so we don't need to run `npm run build` again on deploy
```

Comment:
Do we mean `bundle exec middleman build` rather than `npm run build`?

Reply:
Good catch 👍